scripts/xtensa-build-zephyr.py

```diff
@@ -899,6 +901,60 @@ def build_platforms():
 			symlinks=True, ignore_dangling_symlinks=True, dirs_exist_ok=True)
 
 
+def install_lib(platform, sof_output_dir, abs_build_dir):
+	global signing_key
```

Comment:
should this be a module_signing_key? Is this different from the base firmware key? why is this global?